### ISSUE TITLE
UIControlEventValueChanged for HMSegmentedControl not called on page swipe.

### ISSUE THREAD

**Issue by zeeshankhan** [hons82/THSegmentedPager#24]
Hi,

When we swipe pages, `HMSegmentedControl`'s `UIControlEventValueChanged` event is not getting triggered.

**Comment by hons82** [comment]
Where did you try that?
One of my samples? 
iOS version? Simulator?

**Comment by zeeshankhan** [comment]
I tried it on my application, running with iOS 9.2.
Actually It's an issue of `HMSegmentedControl` class, which does not call `UIControlEventValueChanged ` event.
The only way I see is to manually add these files into my project and fix this thing.
What do you say?

**Comment by hons82** [comment]
In that case it would maybe make sense to fork the HMSegmentedControl and change it there + PR

**Comment by zeeshankhan** [comment]
Yeah, will do it. Once that get merged, will raise another PR here :)

**Comment by hons82** [comment]
Thx... lets c if he's responding in a reasonable amount of time, otherwise, we'll have to create a custom version of `HMSegmenetedControl`, as [I've done in the past too](https://cocoapods.org/?q=hmsegmentedcontrol)
